INFO (30/01/2025 10:48:14): База данных создана (Line: 26 [db.py])
INFO (30/01/2025 10:48:25): Пользователь запросил /tasks/ и сервер вернул 5 тасок! (Line: 17 [task.py])
INFO (30/01/2025 10:48:32): Пользователь попытался найти таску но таски с таким ID (2) не существует! (Line: 57 [task.py])
INFO (30/01/2025 10:48:35): Пользователь получил таску по ID (4) - 4-Создать коллекцию в Postman! (Line: 60 [task.py])
INFO (30/01/2025 10:48:37): Пользователь получил таску по ID (3) - 3-Создать коллекцию в Postman! (Line: 60 [task.py])
INFO (30/01/2025 10:48:39): Пользователь получил таску по ID (1) - 1-Протестировать апи! (Line: 60 [task.py])
INFO (30/01/2025 10:48:42): Пользователь создал новую таску - Создать коллекцию в Postman! (Line: 27 [task.py])
INFO (30/01/2025 10:48:46): Пользователь изменил таску - Протестировать апи! (Line: 46 [task.py])
INFO (30/01/2025 10:49:09): Пользователь изменил таску - Приготовить еды! (Line: 46 [task.py])
INFO (30/01/2025 10:49:15): Пользователь удалил таску по ID (7) (Line: 74 [task.py])
INFO (30/01/2025 10:49:17): Пользователь попытался найти таску но таски с таким ID (7) не существует! (Line: 70 [task.py])
